zoloto/coords.py

```diff
@@ -17,6 +20,13 @@ class Coordinates(NamedTuple):
 
 class ThreeDCoordinates(NamedTuple):
     """
+    Cartesian coordinates, rotated on their side.
+
+    In our coordinate system:
+     - x is side to side in front of the camera
+     - y is up
+     - z is away from the camera
```

Comment:
> Suggestion: It might be nice to add directionality to these

Yes, definitely.

> (Note: I've not cross-referenced as to whether these directions are correct, especially `x`)

No, nor had I, which was why I omitted these. I don't know of anywhere that these are currently documented to compare against nor have a good way to test this to find out. Finding out would be very useful though as we'll need to make the simulator match.

Reply:
#312 renders this moot I believe.